smartautoclicker/src/main/res/values/strings.xml

```diff
@@ -63,6 +63,8 @@
     <string name="dialog_click_config_condition_title">Only if</string>
     <string name="dialog_click_config_delay_after_title">Delay before next click</string>
     <string name="dialog_click_config_duration_hint">ms</string>
+    <string name="dialog_click_config_stop_after_executions">Stop scenario after number of executions</string>
```

Comment:
This might be to long for a small screen, also the formulation is maybe too declarative.
Something simplier like "Executions before scenario end" might be better.